src/explorer/LeetCodeTreeDataProvider.ts

```diff
@@ -42,71 +53,75 @@ export class LeetCodeTreeDataProvider implements vscode.TreeDataProvider<LeetCod
                     title: "Sign in to LeetCode",
                 },
             };
+        } else if (!element.isProblem) { // category
+            return {
+                label: element.name,
+                tooltip: this.getSubCategoryTooltip(element),
+                id: `LeetCode.Category::${element.parentId}.${element.id}`,
```

Comment:
Now that we have `element.parentId`, `Data.Now()` prefix seems no longer necessary.

Btw, the TreeItem Generation code here is separated here for better readability, as well as convenience for next PR(store generated TreeItem somewhere in TreeProvider or LeetCodeNode)

Reply:
So you will send out another PR to remove `element.parentId` & `Data.Now()`, right?

Reply:
Data.Now() has already been removed here, which is taken place by element.parentId